dapps/v2Explorer/src/components/QRCode.tsx

```diff
@@ -2,15 +2,15 @@ import React, {useMemo} from 'react';
 import {Svg} from 'react-native-svg';
 import {View, Image, StyleSheet} from 'react-native';
 import {QrCodeUtil} from '../utils/QrCode';
-import WCLogo from '../assets/WCGradientLogo.png';
+import WCIsotype from '../assets/WCIsotype.png';
 
 interface Props {
   uri: string;
   size: number;
   theme?: 'light' | 'dark';
 }
 
-function QrCode({uri, size, theme = 'light'}: Props) {
+function QRCode({uri, size, theme = 'light'}: Props) {
```

Comment:
Same as this comment. I.e. we have a const with lightTheme = 'light' in a utils folder.